tasks/install-certbot.yml

```diff
@@ -1,14 +1,16 @@
 ---
 - name: Install Certbot from repository and related packages
   package:
-    name: "{{ certbot_packages }}"
-    state: "present"
+    name: "{{ certbot_packages + ['cron'] }}"
+    state: present
+
+- name: systemctl disable --now certbot.timer
+  systemd:
+    name: certbot.timer
```

Comment:
Just for curiosity, is that timer originally installed by the package and used to renew certificates?

If yes, shouldn't we use that instead of a cron job?

Reply:
> Just for curiosity, is that timer originally installed by the package and used to renew certificates?

yes

> If yes, shouldn't we use that instead of a cron job?

yes, but didn't want to rewrite the whole role.